sdks/java/io/solace/src/main/java/org/apache/beam/sdk/io/solace/broker/BasicAuthJcsmpSessionService.java

```diff
@@ -39,34 +43,43 @@
  * <p>This class provides a way to connect to a Solace broker and receive messages from a queue. The
  * connection is established using basic authentication.
  */
-public class BasicAuthJcsmpSessionService extends SessionService {
-  private final String queueName;
-  private final String host;
-  private final String username;
-  private final String password;
-  private final String vpnName;
+@AutoValue
+public abstract class BasicAuthJcsmpSessionService extends SessionService {
```

Comment:
Most AutoValue classes in Beam use get/set prefixes, is there a reason this class does not follow the general style?

Reply:
get/set is used for data classes because schemas rely on the get/set prefix to detect fields.

Other classes, such as Beam PTransforms, don´t use get/set prefixes.

I just find it less verbose using the property names directly.

Reply:
SGTM, non-prefixed properties occur in ~18% of AutoValue class definitions in Beam.